src/main/java/org/xstefank/webhook/WebHookEndpoint.java

```diff
@@ -44,14 +50,27 @@ public void processPullRequest(JsonNode payload) {
     }
 
     private static FormatConfig readConfig() {
+        String url = Utils.readTemplateConfigUrlFromProperties(System
+                .getProperty(Utils.JBOSS_CONFIG_DIR), Utils.CONFIG_FILE);
```

Comment:
also it would be appreciated to allow users to pass the URL through system propery